File: ExotelAPI/Python_Samples/sendansms.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

from pprint import pprint
import requests

from settings import sid, token


def send_message(sid, token, sms_from, sms_to, sms_body):
    return requests.post('https://twilix.exotel.in/v1/Accounts/{sid}/Sms/send.json'.format(sid=sid),
        auth=(sid, token),
        data={
            'From': sms_from,
            'To': sms_to,
            'Body': sms_body
        })


if __name__ == '__main__':
    # 'From' doesn't matter; For transactional, this will be replaced with your SenderId;
    # For promotional, this will be ignored by the SMS gateway
    # Incase you are wondering who Dr. Rajasekhar is http://en.wikipedia.org/wiki/Dr._Rajasekhar_(actor)
    r = send_message(sid, token,
        sms_from='09243422233',  # sms_from='8808891988',
        sms_to='09489244684', # sms_to='9052161119',
        #sms_body='Hi Karthik, Your house tour with HostMe has been allocated and confirmed for your Order ID. Your cab will pick you up. Amount  of Rs. 2999/- has been charged through your PayTM wallet. For further assistance please call 9998854321')
        sms_body='Hi Karthik, Your rent house tour slot has been allocated and confirmed for your Order ID: Uber booked with Bill Amount  2999. For further assistance please call 9998854321')
    print r.status_code
    pprint(r.json())

File: ExotelAPI/PHP_Samples/sendansms.php
<?php

$post_data = array(
    // 'From' doesn't matter; For transactional, this will be replaced with your SenderId;
    // For promotional, this will be ignored by the SMS gateway
    'From'   => '09243422233', 
    'To'    => '09489244684',
    'Body'  => 'Hi Karthik, Your rent house tour slot has been allocated and confirmed for your Order ID: Uber booked with Bill Amount  3999. For further assistance please call 9998854321', //Incase you are wondering who Dr. Rajasekhar is http://en.wikipedia.org/wiki/Dr._Rajasekhar_(actor)
);
 
$exotel_sid = "poulettikka"; // Your Exotel SID - Get it from here: http://my.exotel.in/Exotel/settings/site#api-settings
$exotel_token = "2057a5674069a46f03efa864f4d54831907c0718"; // Your exotel token - Get it from here: http://my.exotel.in/Exotel/settings/site#api-settings
 
$url = "https://".$exotel_sid.":".$exotel_token."@twilix.exotel.in/v1/Accounts/".$exotel_sid."/Sms/send";
 
$ch = curl_init();
curl_setopt($ch, CURLOPT_VERBOSE, 1);
curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_POST, 1);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
curl_setopt($ch, CURLOPT_FAILONERROR, 0);
curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, 0);
curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($post_data));
 
$http_result = curl_exec($ch);
$error = curl_error($ch);
$http_code = curl_getinfo($ch ,CURLINFO_HTTP_CODE);
 
curl_close($ch);
 
print "Response = ".print_r($http_result);
//"https://poulettikka:2057a5674069a46f03efa864f4d54831907c0718@twilix.exotel.in/v1/Accounts/poulettikka/Sms/send?From=09243422233&To=09489244684&Body=TestMMSMSM
?>


